src/pages/ClientPage/style.css

```diff
@@ -3,3 +3,6 @@ body {
   min-height: 100vh;
   background-color: #f0f2f5 !important;
 }
+.ant-btn {
+  position: absolute;
+}
```

Comment:
this might break **a lot** of other buttons in the app, make sure that all other buttons in the other pages are fine